R/Lproj.R

```diff
@@ -197,7 +197,8 @@ create_Lproj <- function(spec, mf_model,
     dplyr::inner_join(
       dplyr::select(mf_model, spectrum_region_code, sex, age_group1 = age_group, area_id, idx1 = idx),
       by = c("spectrum_region_code", "sex", "age_group1"),
-      multiple = "all"
+      multiple = "all",
+      relationship = "many-to-many"
```

Comment:
New warnings coming from dplyr related to many-to-many relationships. See https://www.tidyverse.org/blog/2023/03/dplyr-1-1-1/#revisiting-multiple-matches . I've added these where CI was throwing a warning. @rtesra can you check that these are where you expect them to be?

Reply:
Thanks — I've just reviewed these. All look good and as expected.